eras/allegra/impl/testlib/Test/Cardano/Ledger/Allegra/Era.hs

```diff
@@ -19,6 +20,7 @@ class
   , AllegraEraTxBody era
   , AllegraEraTxAuxData era
   , AllegraEraScript era
+  , NativeScript era ~ Timelock era
```

Comment:
As I've mentioned on our call, this is not a valid constraint, since in Dijkstra the type for `NativeScript` will change and we'd no longer be able to use `AllegraEraTest`.
```suggestion
```